ktor-server/ktor-server-core/jvm/src/io/ktor/server/engine/EnvironmentUtilsJvm.kt

```diff
@@ -73,7 +73,24 @@ internal actual fun ApplicationEnvironmentBuilder.configurePlatformProperties(ar
         ?: ApplicationEnvironment::class.java.classLoader
 }
 
-internal actual fun getConfigFromEnvironment(): ApplicationConfig = System.getProperties()
-    .toMap()
-    .filterKeys { (it as String).startsWith("ktor.") }
-    .let { env -> MapApplicationConfig(env.map { it.key as String to it.value as String }) }
+internal actual fun getKtorEnvironmentProperties(): List<Pair<String, String>> = buildList {
```

Comment:
Environment Properties :D

Reply:
Could you add a note in KDoc for the common declaration of the values used on what platforms? :)

Reply:
It's already there: https://github.com/ktorio/ktor/blob/ca0de5a707d578bdc6be4d3d927ac1a95e4d521d/ktor-server/ktor-server-core/common/src/io/ktor/server/engine/CommandLine.kt#L129-L131